.github/scripts/run_e2e_test.sh

```diff
@@ -1,21 +1,86 @@
-#!/bin/bash
-
-set -eu
-
-while getopts t: opt
-do
-  case $opt in
-    t)
-      export TEST_CASE=$OPTARG;;
-    \?)
-      echo "Invalid option: -$OPTARG"
-      exit 1
-      ;;
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function set_randomized_test_params {
+  # This is arbitrary.
+  MAX_VALIDATOR_COUNT=20
+  VALIDATOR_COUNT=$(shuf -i "${MIN_VALIDATOR_COUNT}"-"${MAX_VALIDATOR_COUNT}" -n 1)
+  # Assumes there is at least one reserved seat for validators.
+  RESERVED_SEATS=$(shuf -i 1-"${VALIDATOR_COUNT}" -n 1)
+  NON_RESERVED_SEATS=$((${VALIDATOR_COUNT} - ${RESERVED_SEATS}))
+}
+
+function usage {
+    cat << EOF
+Usage:
```

Comment:
it is not clear whether we need to provide both --reserved-seats and --non-reserved-seats from description

Reply:
in particular it looks like cli allows to specify only one of them but the script will them only if both of them are not empty at once

Reply:
Added clarification to `usage`.